actors/builtin/network.go

```diff
@@ -29,3 +33,15 @@ func init() {
 		panic(fmt.Sprintf("epoch duration %d does not evenly divide one hour (%d)", EpochDurationSeconds, SecondsInHour))
 	}
 }
+
+// Quality multiplier for committed capacity (no deals) in a sector
+var QualityBaseMultiplier = big.NewInt(10)
```

Comment:
Nit: @zixuanzh  can we rename this to something other than "multiplier", since it's only ever used in a denominator.

Reply:
any suggestion? maybe we should split this into CommittedCapacityMultiplier =10 and then QualityBase = 10? We can make this 100 instead of 10 too to have another bit of precision but we dont need it atm.